tests/Dfe.PlanTech.AzureFunctions.UnitTests/Mappers/RecommendationChunkMapperTests.cs

```diff
@@ -266,10 +270,19 @@ private static void ValidateContent(string[] contentIds, RecommendationChunkDbEn
         var fields = new Dictionary<string, object?>()
         {
             ["answers"] = WrapWithLocalisation(answers),
-            ["content"] = WrapWithLocalisation(contents),
-            ["csLink"] = WrapWithLocalisation(csLink)
+            ["content"] = WrapWithLocalisation(contents)
         };
 
+        if (csLink != null)
+        {
+            var csLinkFields = new Dictionary<string, object?>
+                {
+                    ["url"] = WrapWithLocalisation(csLink.Url),
+                    ["linkText"] = WrapWithLocalisation(csLink.LinkText)
+                };
+            fields["csLink"] = CreatePayload(csLinkFields, csLink.Sys.Id);
+        }
```

Comment:
You just need to reference the Id as the referenced content itself doesn't need to come back here
```suggestion
        if (csLinkId != null)
        {
            var link = new CmsWebHookSystemDetailsInnerContainer()
            {
                Sys = new CmsWebHookSystemDetailsInner()
                {
                    Id = csLinkId
                }
            };
            fields["csLinkId"] = WrapWithLocalisation(link);
        }
```